authentication/pipeline/user.py

```diff
@@ -187,6 +187,9 @@ def require_touchstone_login(
         backend (social_core.backends.base.BaseAuth): the backend being used to authenticate
         flow (str): the type of flow (login or register)
     """
+    # As first step in pipeline, stop a hijacking admin from going any further
```

Comment:
To keep things clean, can you split this out to it's own pipeline step instead?

Reply:
Can you write a test for this too?

Reply:
✔️